opencog/atoms/proto/README.md

```diff
@@ -0,0 +1,70 @@
+
+ProtoAtoms
+==========
+ProtoAtoms are a partially-implemented, incompletely thought-out
+concept for a way of associating arbitrary generic values with some
+given atom.  ProtoAtoms are like atoms, except that they lack a TV,
+an AV, and cannot be stored in the atomspace.  This makes them smaller,
+lighter and more efficient.
+
```

Comment:
Almost everything that the rest of this README says is wrong; I will try to update it.